data/morrow-base/morrow.yml

```diff
@@ -79,6 +88,9 @@
   components:
   - player_config
   - input
+  - character:
+      class_level:
+        warrior: 1
```

Comment:
Remove this.  Probably should just do `level` in `morrow:char`, then specify classes in the player creation.  The default value in `level` will be over-written by `update_char_level()`

Reply:
Didn't work because we `update_char_resources()` in `spawn()`, so if this is removed we get an exception when spawning the player.